Analyzer/include/Histo.h

```diff
@@ -6,6 +6,8 @@
 #include <iostream>
 #include <string>
 
+#include "TH3D.h"
```

Comment:
Was this needed? Just curious doesn't really matter either way.

Reply:
When cleaning up some includes (due to switch away from `TLorentzVector`) I needed to explicitly include `TH3D.h` as the methods were no longer getting included from wherever they were coming from before.